rdt/hyper_transformer.py

```diff
@@ -792,10 +796,54 @@ def fit_transform(self, data):
         self.fit(data)
         return self.transform(data)
 
+    def create_anonymized_columns(self, num_rows, column_names):
+        """Create the anonymized columns for this ``HyperTransformer``.
+
+        Generate the anonymized and text based columns that use regex expressions.
+
+        Args:
+            num_rows (int):
+                Number of rows to be created. Must be an integer greater than 0.
+            column_names (list):
+                List of column names to be created.
+
+        Returns:
+            pandas.DataFrame:
+                A data frame with the newly generated columns of the size ``num_rows``.
+        """
+        self._validate_fitted()
+
+        if not isinstance(num_rows, int) or num_rows <= 0:
+            raise Error("Parameter 'num_rows' must be an integer greater than 0.")
+
+        unknown_columns = self._subset(column_names, self._input_columns, not_in=True)
+        if unknown_columns:
+            raise Error(
```

Comment:
This is out of scope for this PR, but I just wanted to leave a comment - `Error` seems a little generic. Maybe something like `HyperTransformerError` would be more informative?